DOCUMENTATION.md

```diff
@@ -439,32 +439,33 @@ You can only get the status of messages that are 7 days old or newer.
+|Failed|This covers all failure statuses:<br>- `permanent-failure` - "The provider could not deliver the message because the email address or phone number was wrong. You should remove these email addresses or phone numbers from your database. You’ll still be charged for text messages to numbers that do not exist."<br>- `temporary-failure` - "The provider could not deliver the message after trying for 72 hours. This can happen when the recipient's inbox is full or their phone is off. You can try to send the message again. You’ll still be charged for text messages to phones that are not accepting messages."<br>- `technical-failure` - "Your message was not sent because there was a problem between Notify and the provider.<br>You’ll have to try sending your messages again. You will not be charged for text messages that are affected by a technical failure."|
 
 ## Status - text only
 
 |Status|Information|
 |:---|:---|
-|Pending|GOV.UK Notify received a callback from the provider but the device has not yet responded. Another callback from the provider determines the final status of the notification.|
-|Sent|The text message was delivered internationally. This only applies to text messages sent to non-UK phone numbers. GOV.UK Notify may not receive additional status updates depending on the recipient's country and telecoms provider.|
+|Pending|GOV.UK Notify is waiting for more delivery information.<br>GOV.UK Notify received a callback from the provider but the recipient's device has not yet responded. Another callback from the provider determines the final status of the notification.|
+|Sent / Sent internationally|The message was sent to an international number. The mobile networks in some countries do not provide any more delivery information. The GOV.UK Notify client API returns this status as `sent`. The GOV.UK Notify client app returns this status as `Sent internationally`.|
 
 ## Status - letter
 
 |Status|information|
 |:---|:---|
 |Failed|The only failure status that applies to letters is `technical-failure`. GOV.UK Notify had an unexpected error while sending to our printing provider.|
-|Accepted|GOV.UK Notify is printing and posting the letter.|
-|Received|The provider has received the letter to deliver.|
+|Accepted|GOV.UK Notify has sent the letter to the provider to be printed.|
+|Received|The provider has printed and dispatched the letter.|
 
 ## Status - precompiled letter
 
 |Status|information|
 |:---|:---|
-|Pending virus check|GOV.UK Notify virus scan of the precompiled letter file is not yet complete.|
-|Virus scan failed|GOV.UK Notify virus scan has identified a potential virus in the precompiled letter file.|
+|Pending virus check|GOV.UK Notify has not completed a virus scan of the precompiled letter file.|
```

Comment:
should be `pending-virus-check`, `virus-scan-failed` and `validation-failed`

Reply:
Do we need to phrase it as it would look in the API? I thought these tables were non-code formatted as they are descriptions rather than code blocks.

Reply:
I thought these tables were non-code formatted too – because they provide information about statuses, rather than instructions/method/arguments.

The statuses are formatted as code in the _Get the status of multiple messages_ section: https://docs.notifications.service.gov.uk/python.html#get-the-status-of-multiple-messages

Reply:
Becca confirmed PR is fine on slack at 1056am on 23/01/2019